pkg/upgraders/osd/upgrader.go

```diff
@@ -126,9 +126,14 @@ func PreClusterHealthCheck(c client.Client, cfg *osdUpgradeConfig, scaler scaler
 	if err != nil {
 		return false, err
 	}
+
 	desired := upgradeConfig.Spec.Desired
 	if upgradeCommenced {
-		logger.Info(fmt.Sprintf("ClusterVersion is already set to Channel %s Version %s, skipping %s", desired.Channel, desired.Version, upgradev1alpha1.UpgradePreHealthCheck))
+		if len(desired.Channel) > 0 && len(desired.Version) > 0 {
```

Comment:
can we just reuse this in some way instead of copy the same block multiple times?

Reply:
Done.. Moved the logging part to clusterversion package then the upgrader..